subscribers:
  camera_reading:
    topic: /camera/color/image_raw
    queue_size: 1

publishers:
  bounding_boxes:
    topic: /darknet_ros/bounding_boxes
    queue_size: 1
    latch: false

image_view:
  enable_opencv: false
  wait_key_delay: 1
  enable_console_output: false
